packages/govern-console/src/containers/NewProposal/NewProposal.tsx

```diff
@@ -180,9 +180,9 @@ const NewProposal: React.FC<NewProposalProps> = ({ onClickBack, ...props }) => {
     }
   }, [daoList]);
 
-  const proposal = React.useMemo(() => {
+  const proposal = React.useMemo(() => {  
     if (daoDetails) {
-      executor = daoDetails.executor.address;
+      executor.current = daoDetails.executor.address;
       return new Proposal(daoDetails.queue.address, proposalOptions);
     }
   }, [daoDetails]);
```

Comment:
Actually you don't need to add executor ref, only adding [daoDetails, executor] was required to recompute on change of executor

Reply:
@reddyismav

Reply:
We will take this up after todays demo